### DISCUSSION TITLE
Not sure if new feature or somehow already possible - set start image or use multuple prompts in a row

### DISCUSSION THREAD

**Discussion by PintJaguar** [BalintKomjati/smooth-infinite-zoom#2] [Q&A]
Hello, i would love to use this Notebook for creating a music video and tell a story, but for this i would either need the possibility to also set a starting picture (which would be the last frame of a previous video) and then stitch them together in a video editing tool OR get the possibility to use multiple prompts within this tool, starting from a specific frame - basically like in deforums animation prompts.

**Comment by BalintKomjati** [comment]
Hi PintJaguar! sorry for not coming back before. We exchanged about this already on reddit, right? Did you manage to do that? Uploading manually an init_image should be very easy!

**Comment by gordingus** [comment]
Seconding the request for the ability to schedule prompts! Although the original solution is a good one too
